packages/legend-graph/src/models/protocols/pure/v1/engine/generation/V1_FileGenerationInput.ts

```diff
@@ -16,17 +16,17 @@
 
 import { createModelSchema, optional, primitive, raw } from 'serializr';
 import { SerializationFactory } from '@finos/legend-shared';
-import type { V1_PureModelContextData } from '../../model/context/V1_PureModelContextData';
-import { V1_pureModelContextDataPropSchema } from '../../transformation/pureProtocol/V1_PureProtocolSerialization';
+import { V1_pureModelContextTextPropSchema } from '../../transformation/pureProtocol/V1_PureProtocolSerialization';
 import { PureClientVersion } from '../../../../../../graphManager/GraphManagerUtils';
+import type { V1_PureModelContextText } from '../../model/context/V1_PureModelContextText';
 
 export class V1_GenerateFileInput {
   clientVersion?: string | undefined;
-  model: V1_PureModelContextData;
+  model: V1_PureModelContextText;
```

Comment:
why this? why don't we do `V1_PureModelContext`?

Reply:
I think we may want to precisely use PureModelContextText here; otherwise, this would mean that we need a mechanism (a prop schema) to deserialize from any unknown subtype of PureModelContext.

Reply:
See comment below, I think this should be `V1_PureModelContext` to align with engine.